pages/agent/v2/debian.md.erb

```diff
@@ -30,7 +30,7 @@ sudo apt-get install -y apt-transport-https dirmngr
 Now you can add our signed apt repository:
 
 ```shell
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 32A37959C2FA5C3C99EFBC32A79206696452D198
+sudo apt-key adv --keyserver hkps://keyserver.ubuntu.com:80 --recv-keys 32A37959C2FA5C3C99EFBC32A79206696452D198
```

Comment:
I just gave this a test, and looks like we need to drop the `:80` because `hkps` is a different port. This should do it:

```suggestion
sudo apt-key adv --keyserver hkps://keyserver.ubuntu.com --recv-keys 32A37959C2FA5C3C99EFBC32A79206696452D198
```